app/views/documents/index.html.erb

```diff
@@ -24,7 +24,9 @@
         <%= govspeak_to_html t("documents.index.search_results.guidance") %>
       </div>
     <% else %>
-      <%= render "documents/index/results" %>
+      <div class="results-table-wrapper">
```

Comment:
We won't need the additional wrapper as the overwrite should be applied at the project level.